exe/bolt_catalog

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'bolt'
+require 'bolt/catalog'
+require 'json'
+
+# This accepts a catalog request on stdin:
+# { "code_ast": "JSON serialized Puppet AST",
+#   "code_string": "String of code, ignored if AST is provided,
+#   "modulepath": "Array of directories to use as the modulepath for catalog compilation.
+#   "target": {
+#   "name": "the name of the node usually fqdn fro url",
+#   "facts": "Hash of facts to use for the node",
+#   "variables": "Hash of variables to use for compilation"
+#   }
+
+command = ARGV[0]
+if command == "parse"
+  code = File.open(ARGV[1], &:read)
+  puts Bolt::Catalog.new.generate_ast(code)
+elsif command == "compile"
+  request = if ARGV[1]
+              File.open(ARGV[1]) { |fh| JSON.parse(fh) }
+            else
+              JSON.parse(STDIN)
```

Comment:
This didn't work, maybe `STDIN.read`.

Reply:
annoying it was load but rubocop didn't like it

Reply:
Seems rubocop wants you to be explicit about reading from a stream, so `JSON.parse(STDIN.read)` rather than `JSON.load(STDIN)`: https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_security.md#securityjsonload. I'm having trouble identifying why it's a real security concern though.